src/client/package.json

```diff
@@ -38,18 +38,22 @@
     "finsemble:build": "cross-env TARGET=finsemble vite build",
     "format": "prettier --write README.md \"src/**/*.{js,jsx,ts,tsx,json,md}\"",
     "generateCod": "hydra-web-codegen -i trading-gateway.hyer -o src/generated",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook -o ./dist/storybook"
+    "storybook": "start-storybook -s ./public -p 6006",
+    "build-storybook": "build-storybook"
+  },
+
+  "cypress-cucumber-preprocessor": {
+    "nonGlobalStepDefinitions": true
   },
   "dependencies": {
-    "@adaptive/hydra-platform": "^3.40.0",
+    "@adaptive/hydra-platform": "^3.32.0",
```

Comment:
There are a lot of dependency changes here which are unrelated to your work, perhaps some merge resolution issues. Please go through and revert any irrelevant packages to what is currently in `master`